cli/Valet/Pecl.php

```diff
@@ -313,7 +313,7 @@ function fix(){
         // Check if pear config is set correctly as per:
         // https://github.com/kabel/homebrew-core/blob/2564749d8f73e43cbb8cfc449bca4f564ac0e9e1/Formula/php%405.6.rb
         // Brew installation standard.
-        foreach (Brew::SUPPORTED_PHP_FORMULAE as $phpVersion => $brewname) {
+        foreach (PhpFpm::SUPPORTED_PHP_FORMULAE as $phpVersion => $brewname) {
             output("Checking php $phpVersion...");
 
             $pearConfigPath = "/usr/local/etc/php/$phpVersion/pear.conf";
```

Comment:
This one also needs to be changed to: `"/usr/local/etc/valet-php`

Reply:
But should this be only for PHP 5.6 and few more? Or all the versions will be there?

Reply:
Yeah for all, we moved the config path of all valet-php versions to ensure there will be no conflict with local installs.